packages/git-raw-commits/index.js

```diff
@@ -40,6 +41,7 @@ function gitRawCommits(options) {
   var isError = false;
 
   var child = execFile('git', args, {
```

Comment:
If you pass `options.cwd`, would it take it as an option already?

Reply:
I could pluck `cwd` from options, too. I thought it would be wise to keep git-specific options separated from ones geared towards the mode of execution / `process.execFile`

Reply:
Maybe rename it to something like `gitOpts` then :)